trino/dbapi.py

```diff
@@ -584,66 +603,22 @@ def describe(self, sql: str) -> List[DescribeOutput]:
 
         return list(map(lambda x: DescribeOutput.from_row(x), result))
 
-    def genall(self):
-        return self._query.result
+    def genall(self) -> Any:
```

Comment:
```suggestion
    def genall(self) -> Optional[trino.client.TrinoResult]:
```

Reply:
mypy throws error then:
```trino/dbapi.py:615: error: Argument 1 to "list" has incompatible type "Optional[TrinoResult]"; expected "Iterable[List[Any]]"  [arg-type]```

changing to `Iterable[List[Any]]` throws error elsewhere, and so on